content/docs/api-reference/make_checkpoint.md

```diff
@@ -29,8 +29,8 @@ logger). This function can be called by the code in stages executed by
 1. Check that the `DVC_ROOT` env var is set. It means this code is being
    executed via `dvc exp run`, and it contains the location to the correct
    `.dvc/` directory for this experiment (which can vary when `-j` is used)
-2. Creates an empty `$DVC_ROOT/tmp/DVC_CHECKPOINT` signal file so DVC knows that
-   a checkpoint should be captures now.
+2. Creates an empty `$DVC_ROOT/.dvc/tmp/DVC_CHECKPOINT` signal file so DVC knows
+   that a checkpoint should be captured now.
```

Comment:
Thanks for the typo fix!

Doesn't `$DVC_ROOT` point to `.dvc/` already?

Reply:
Ah I see not, per https://github.com/iterative/dvc-checkpoints-mnist/blob/6e215a7d9b06e34608d1facccd42b46a2de1f3b7/train.py#L111-L113. OK cool!